wagtail/admin/checks.py

```diff
@@ -166,6 +166,72 @@ def check_panels_in_model(cls, context="model"):
     return errors
 
 
+@register("panels")
+def panel_type_check(app_configs, **kwargs):
+    from wagtail.models import get_page_models
+    from wagtail.snippets.models import get_snippet_models
+
+    errors = []
+
+    for cls in get_page_models():
+        errors += traverse_edit_handlers(cls.get_edit_handler())
+
+    for cls in get_snippet_models():
+        if hasattr(cls, "panels"):
+            for panel in cls.panels:
+                # Check if panel is MultiFieldPanel
+                if hasattr(panel, "children"):
+                    for panel_child in panel.children:
+                        errors += traverse_edit_handlers(panel_child.bind_to_model(cls))
+                errors += traverse_edit_handlers(panel.bind_to_model(cls))
+
+    return errors
+
+
+def traverse_edit_handlers(edit_handler):
+    errors = []
+
+    try:
+        for child in edit_handler.children:
+            errors += traverse_edit_handlers(child)
+    except AttributeError:
+        panel_field_error = check_panel_field_valid(edit_handler)
+        if panel_field_error:
+            errors.append(panel_field_error)
+
+    return errors
+
+
+def check_panel_field_valid(edit_handler):
+    from wagtail.admin.panels import CommentPanel, TabbedInterface
+
+    if isinstance(edit_handler, (CommentPanel, TabbedInterface)):
```

Comment:
```suggestion
    from wagtail.admin.panels import CommentPanel, HelpPanel, TabbedInterface

    if isinstance(edit_handler, (CommentPanel, HelpPanel, TabbedInterface)):
        pass
```

`HelpPanel` is needed, otherwise you get a warning for these.



```
web       | System check identified some issues:
web       | 
web       | WARNINGS:
web       | recipes.RecipePage: (wagtailadmin.W005) RecipePage does not have a field named 'None', but a HelpPanel is pointing to it
web       | 	HINT: Check whether the field exists on the model and ensure it has been typed correctly
```